Update User Guide - Legend filtering for WMS layers (#11026)
geosolutions-it/MapStore2: docs/user-guide/toc.md

* Expand <img src="../img/button/expand-all-child-nodes.jpg" class="ms-docbutton"/> or collapse <img src="../img/button/collapse-all-child-nodes.jpg" class="ms-docbutton"/> the layer legend.

!!! note

>     The WMS layer legend can be dynamically filtered based on the [Filter](filtering-layers.md#filter-types) applied to the layer and the bounding box corresponding to the current map extent, provided that the **Dynamic legend** option is enabled in the [Layer Settings](layer-settings.md#display).

Reviewer: ```suggestion
    The WMS layer legend can dynamically update based on the [Filter](filtering-layers.md#filter-types) applied to the layer and/or the current map viewport if the **Dynamic legend** option is enabled in the [Layer Settings](layer-settings.md#display).
```
